Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx

```diff
@@ -363,16 +363,15 @@ void ITSDCSParser::pushToCCDB(EndOfStreamContext* ec)
   auto image = o2::ccdb::CcdbApi::createObjectImage(&mConfigDCS, &info);
   info.setFileName(filename);
 
-  if (ec) { // send to ccdb-populator wf only if there is an EndOfStreamContext
-    LOG(info) << "Class Name: " << class_name << " | File Name: " << filename
-              << "\nSending to ccdb-populator the object " << info.getPath() << "/" << info.getFileName()
-              << " of size " << image->size() << " bytes, valid for "
-              << info.getStartValidityTimestamp() << " : "
-              << info.getEndValidityTimestamp();
+  // Send to ccdb-populator wf
+  LOG(info) << "Class Name: " << class_name << " | File Name: " << filename
+            << "\nSending to ccdb-populator the object " << info.getPath() << "/" << info.getFileName()
+            << " of size " << image->size() << " bytes, valid for "
+            << info.getStartValidityTimestamp() << " : "
+            << info.getEndValidityTimestamp();
 
-    ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "DCS_CONFIG", 0}, *image);
-    ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "DCS_CONFIG", 0}, info);
-  }
+  pc.outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "DCS_CONFIG", 0}, *image);
+  pc.outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "DCS_CONFIG", 0}, info);
```

Comment:
please defines these outputs in the getITSDCSParserSpec as `Lifetime::Sporadic`, see e.g. https://github.com/AliceO2Group/AliceO2/blob/ab0533d875fa6920665ee6ba3aafe43e8081c5b2/Detectors/TOF/calibration/testWorkflow/TOFDCSConfigProcessorSpec.h#L142-L143. This is a general rule for all messages to the CCDB populator.